src/build123d/geometry.py

```diff
@@ -126,6 +126,51 @@ def _parse_intersect_args(*args, **kwargs):
     return axis, plane, vector, location, shape
 
 
+def _parse_location_args(*args, **kwargs):
+    position, orientation, ordering, plane, location, top_loc, gp_trsf = (None,) * 7
```

Comment:
This line isn't required as the `pop` method will either assign the `kwargs["position"]` or if there is no "position" key, `None`.

Reply:
Good point, I forgot about the functionality of the dict pop default.